content/docs/how-we-work/release-versioning.md

```diff
@@ -9,3 +9,11 @@ description: We give attention to release versioning as it is the practice of as
 We give attention to release versioning as it is the practice of assigning unique identifiers or version numbers to software releases, indicating changes, updates, or improvements made to the software. These version numbers typically follow a specific format or convention, allowing developers, users, and other stakeholders to understand the significance of each release and track the evolution of the software over time.
 
 We adhere to the [Semantic Versioning 2.0.0](https://semver.org/) format, in line with Drupal's practice, for release versioning. This method assigns version numbers to software releases, with each number representing major, minor, and patch updates. Following this standardized approach ensures clarity and consistency in communicating changes to our products, facilitating seamless collaboration and understanding among our team members and users.
+
+## Release Planning
+
+Release planning involves collaborative efforts between teams and product owners to determine the "how," "when," and "what" of upcoming product increments. Regardless of project size or duration, we collaborate with our project managers for maintaining updated release plans for every project.
```

Comment:
@Yemaneberhan-Lemma I think we need to point out about making releases more frequent, automated and less of a big event. 

Any content along those lines would be good.
CC: @hussainweb